tide/core/scene/ContentFactory.cpp

```diff
@@ -160,19 +160,21 @@ ContentPtr ContentFactory::getContent( const QString& uri )
     return ContentPtr();
 }
 
-ContentPtr ContentFactory::getPixelStreamContent( const QString& uri )
+ContentPtr ContentFactory::getPixelStreamContent( const QString& uri, const StreamType stream, const bool keyboard )
 {
-    return ContentPtr( new PixelStreamContent( uri ));
-}
-
-ContentPtr ContentFactory::getWebbrowserContent( const QString& uri )
-{
-#if TIDE_USE_QT5WEBKITWIDGETS || TIDE_USE_QT5WEBENGINE
-    return ContentPtr( new WebbrowserContent( uri ));
-#else
-    Q_UNUSED( uri );
-    throw std::runtime_error( "Tide was compiled without WebbrowserContent!" );
-#endif
+    if ( stream == WEBBROWSER )
+    {
+        #if TIDE_USE_QT5WEBKITWIDGETS || TIDE_USE_QT5WEBENGINE
```

Comment:
- Leave #if macros indented to the left.
- The idea of passing the StreamType here was also to remove the keyboard bool parameter and decide here to hide it for the whiteboards, which in turns makes the code cleaner in PixelStreamWindowManager.cpp